src/components/CustomMessageSystem/index.js

```diff
@@ -23,7 +23,7 @@ const index = ({text}) => (
       }}>
       <Text
         style={{
-          color: COLORS.blackgrey,
+          color: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the change from `COLORS.blackgrey` to `COLORS.gray400` should be fine as long as `COLORS.gray400` is defined and it fits with the overall theme of your application. Also, ensure that this color change doesn't affect the readability and accessibility of the text.

```diff
-          color: COLORS.blackgrey,
+          color: COLORS.gray400,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->